bridge/client/app/_components/ktb-service-settings-overview/ktb-service-settings-overview.component.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { KtbServiceSettingsOverviewComponent } from './ktb-service-settings-overview.component';
+import { AppModule } from '../../app.module';
+import { ActivatedRoute, convertToParamMap } from '@angular/router';
+import { of } from 'rxjs';
+import { By } from '@angular/platform-browser';
+
+describe('KtbServiceSettingsOverviewComponent', () => {
+  let component: KtbServiceSettingsOverviewComponent;
+  let fixture: ComponentFixture<KtbServiceSettingsOverviewComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [
+        AppModule,
+      ],
+      providers: [
+        {
+          provide: ActivatedRoute,
+          useValue: {
+            paramMap: of(convertToParamMap({
+              projectName: 'sockshop',
+            })),
+          },
+        },
+      ],
+    }).compileComponents();
+    fixture = TestBed.createComponent(KtbServiceSettingsOverviewComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should show create button', () => {
+    // given
+    const creatButton = fixture.debugElement.query(By.css('button'));
```

Comment:
Typo: createButton
Also you extend the test for text "Create service" on the button